frontend/src/pages/adm/AdmMenu.jsx

```diff
@@ -9,6 +9,10 @@ function AdmMenu () {
                 link='/admin/cadastro'
                 text='Cadastro de Administrador'
```

Comment:
Remove this option as it is already covered in the new page?

Reply:
sure. done.